ui/app/components/mount-backend-form.js

```diff
@@ -56,6 +56,17 @@ export default class MountBackendForm extends Component {
     }
   }
 
+  typeChangeSideEffect(type) {
+    if (!this.args.mountType === 'secret') return;
```

Comment:
Are we checking to see if there's no `mountType` or `this.args.mountType !== 'secret'`? If we're checking if there's no mountType, it won't ever equal 'secret'?

Reply:
Good question! Here I essentially want to continue with the side effects *only* if mountType is secret (auth is the other possibility). If it makes more sense to read as below, I'm happy to update it
```
if (this.args.mountType === 'secret') {
  // do stuff
}
```

Reply:
Got it - that makes sense to me! I was reading it like `!this.args.mountType` === 'secrets' but it's really just checking ` (!(this.args.mountType === 'secrets'))`. I think we can keep it as it is! Thanks for clarifying 🙃